src/communication/channelmanager.cpp

```diff
@@ -54,6 +54,8 @@ RetWithError<ChannelItf*> ChannelManager::CreateChannel(uint32_t port)
         return {nullptr, err};
     }
 
+    lock.Unlock();
```

Comment:
maybe also better to use LockGuard and unnamed block?

Reply:
In this case I'd like to  keep as it.
The main reason is the channel variable that's declared only if mChannels doesn't contain the desired port.

It's not a big deal to declare it as nullptr before proposed LockGuard scope.
But I don't see much benefits in this change.